ansible_image_validation/validation-playbooks/per-vm-validation.yaml

```diff
@@ -58,4 +58,34 @@
 - name: Check if blacklisted drivers are blacklisted by modprobe service
   include_tasks: validation-playbooks/blacklisted_drivers_validation.yaml
   ignore_errors: yes
-  when: ansible_os_family == "RedHat"
\ No newline at end of file
+  when: ansible_os_family == "RedHat"
+
+- name: Check the kernel version
```

Comment:
this task is not required since you can get the kernel version from the ansible facts 
check :     "ansible_kernel":  in https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_vars_facts.html